specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHubCluster.json

```diff
@@ -0,0 +1,560 @@
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Event Hubs Cluster resource description returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response describing why the cluster description get operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Event Hubs Cluster"
+        ],
+        "operationId": "Clusters_Patch",
+        "x-ms-examples": {
+          "ClusterPatch": {
+            "$ref": "./examples/Clusters/ClusterPatch.json"
+          }
+        },
+        "description": "Modifies mutable properties on the Event Hubs Cluster. This operation is idempotent.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            },
+            "description": "The properties of the Event Hubs Cluster which should be updated."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Event Hubs Cluster successfully created.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "201": {
+            "description": "Event Hubs Cluster update request accepted.",
+            "schema": {
+              "$ref": "#/definitions/Cluster"
+            }
+          },
+          "202": {
+            "description": "Event Hubs Cluster update request accepted."
+          },
+          "default": {
+            "description": "Event Hubs error response detailing why the Event Hubs Cluster update operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/quotaConfiguration/default": {
+      "patch": {
+        "tags": [
+          "Event Hubs Cluster Configuration"
+        ],
+        "operationId": "Clusters_ConfigurationPatch",
```

Comment:
You could probably rename all of the configuration operations like so.
```
Configuration_Path
```
This would generate a new client named ```Configuration``` so the method names won't collide with what's in the ```Clusters``` client.